org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/ConstList.java

```diff
@@ -16,11 +16,7 @@
 package edu.mit.csail.sdg.alloy4;
 
 import java.io.Serializable;
-import java.util.AbstractList;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.RandomAccess;
+import java.util.*;
```

Comment:
This can potentially cause problems if classes are added to java.util. We also have autoformatting and auto organize imports in place so these list should **not** be maintained by hand.